ansible/host_vars/test-iinthecloud-ibmi72-ppc64_be-1

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
this can be removed before landing, and I'll directly put the secrets in nodejs/secrets

Reply:
I've added the secrets to the test machines to the inventory in the secrets repository. I'll push an update here after lunch to remove this file and then run the ansible script on the second test node (https://ci.nodejs.org/computer/test-iinthecloud-ibmi72-ppc64_be-2/).

Reply:
Removed the host_vars.